### ISSUE TITLE
Starred tickets - notification on update

### ISSUE THREAD

**Issue by bashtoni** [BashtonLtd/rrequest#73]
Sometimes a customer or staff member might be waiting on a reply to a specific ticket, but working on other things.

It would be nice to have the ability to star one or more tickets.  When these tickets are updated, a notification should be displayed to the user.  It would be great if this was a desktop notification.
